sps30-i2c/sps30.h

```diff
@@ -46,6 +46,14 @@ extern "C" {
 #define SPS30_MEASUREMENT_DURATION_USEC 1000000
 /* 50ms delay after resetting the sensor */
 #define SPS30_RESET_DELAY_USEC 50000
+/** SPS30_DEVICE_STATUS_FAN_ERR - The fan is switched on but not running */
+#define SPS30_DEVICE_STATUS_FAN_ERR(device_status) ((device_status) & (1 << 4))
+/** SPS30_DEVICE_STATUS_LASER_ERR - The laser current is out of range */
+#define SPS30_DEVICE_STATUS_LASER_ERR(device_status) \
+    ((device_status) & (1 << 5))
+/** SPS30_DEVICE_STATUS_FAN_SPEED_ERR - The fan speed is out of range */
+#define SPS30_DEVICE_STATUS_FAN_SPEED_ERR(device_status) \
```

Comment:
The fan speed flag is not an error but a warning according to the datasheet:
![sps30-device-status-register](https://user-images.githubusercontent.com/1435346/88262199-03f2f100-ccc8-11ea-8d8e-0a8ffed0da98.png)

Reply:
Fixed.